options/base_options.py

```diff
@@ -71,6 +71,9 @@ def initialize(self, parser):
         parser.add_argument('--f_s_semantic_threshold',default=1.0,type=float,help='threshold of the semantic classifier loss below with semantic loss is applied')
         parser.add_argument('--f_s_all_classes_as_one',action='store_true',help='if true, all classes will be considered as the same one (ie foreground vs background)')
         parser.add_argument('--f_s_nf', type=int, default=64, help='# of filters in the first conv layer of classifier')
+        parser.add_argument('--f_s_config_segformer',type=str,default='models/configs/segformer/segformer_config_b0.py',help='path to segforme configuration file')
```

Comment:
`f_s` is for the semantic network at the moment, we need a different option here, as this is the architecture of the Discriminator's projector.
Something like `--D_proj_config` instead.